test/e2e/federated-secret.go

```diff
@@ -179,5 +178,5 @@ func waitForSecretUpdateOrFail(clientset *release_1_3.Clientset, namespace strin
 }
 
 func equivalentSecret(federatedSecret, clusterSecret v1.Secret) bool {
-	return reflect.DeepEqual(clusterSecret, federatedSecret)
+	return util.SecretEquivalent(clusterSecret, federatedSecret)
```

Comment:
nit: It seems that you could get rid of equivalentSecret() and replace all calls to it with util.SecretEquivalent() ?